content/agent/kubernetes/daemonset_setup.md

```diff
@@ -238,29 +260,37 @@ Learn more about this in [the Docker log collection documentation][9].
 
 ### Trace Collection
 
-To enable [Trace collection][10] with your DaemonSet:
+To enable trace collection with your DaemonSet:
```

Comment:
This section is super important. It must stay here but I'd say to add a link for all languages? So:
* keep the section here
* in all languages (such as https://docs-staging.datadoghq.com/gus/agent-apm-2/tracing/languages/java/) change `see additional documentation for tracing Docker applications` in `see additional documentation for tracing Docker applications and Kubernetes` with a link to this section

In this way a random team that has the Agent already deployed can just open the language section, see Kubernetes needs to rebuild the container, install the tracer, add the following config, and they're done.

Makes sense?